apps/ci/ci-install.sh

```diff
@@ -26,8 +26,8 @@ CTYPE=Release
 CSCRIPTS=$1
 CSERVERS=ON
 CTOOLS=ON
-CSCRIPTPCH=OFF
-CCOREPCH=OFF
+CSCRIPTPCH=ON
+CCOREPCH=ON
```

Comment:
No, why did you change that??

It needs to be set to OFF because PCH (precompiled headers) are optional. Travis must not compile them with PCH on because else it is possible that modules/scripts are missing headers and then compilation failed. This is something me, rochet and most notably @Stoabrogga had to deal with. Unless I'm making a mistake, of course.

Reply:
This commit only test witch pch ...

Reply:
ok so it was only temporary?